README.adoc

```diff
@@ -83,6 +60,10 @@ compiler currently runs on the following platforms:
 Other operating systems for the processors above have not been tested, but
 the compiler may work under other operating systems with little work.
 
+❗ From OCaml 5.0 onwards, native compilation is available only on 64-bit
```

Comment:
Maybe
```suggestion
❗ From OCaml 5.0 onwards, native compilation is available only on some 64-bit
```
or (`a few`)

Reply:
That makes it sound as though it's fewer 64-bit systems from OCaml 5.0, though? Or perhaps drop the "From OCaml 5.0 onwards," as well?